piperider_cli/profiler/schema.json

```diff
@@ -318,6 +319,76 @@
                 }
               }
             },
+            "piperider_assertion_result": {
+              "oneOf": [
+                {
+                  "type": "null"
+                },
+                {
+                  "title": "PipeRiderAssertionResult",
+                  "type": "object",
+                  "required": [
+                    "tests",
+                    "columns"
+                  ],
+                  "additionalProperties": false,
+                  "properties": {
+                    "tests": {
+                      "type": "array",
+                      "items": {
+                        "$ref": "#/definitions/assertion_test"
```

Comment:
Due to the `piperider_assertion_result` and `dbt_assertion_result` are used for backward  compatible. Can we not reference the latest `assertion_test`?
Perhaps we can add a new definition called `legacy_assertion_test` and the values of this definition could be the old schema values.

Reply:
sure, it sounds more reasonable!